src/pages/SignUp/SignUp.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from "react";
+import ConfirmForm from "./ConfirmForm";
+import SignupForm from "./SignupForm";
+import Success from "./Success";
+import { SignupState } from "./types";
+
+export default function SignUp() {
+  const [state, setState] = useState<SignupState>({ type: "init" });
+
+  const content = (() => {
+    if (state.type === "init") {
+      return <SignupForm setSignupState={setState} />;
+    }
+
+    if (state.type === "success") {
+      return <Success userType={state.userType} />;
+    }
+
+    return (
+      <ConfirmForm
+        userType={state.userType}
+        codeRecipientEmail={state.codeRecipientEmail}
+        setSignupState={setState}
+      />
+    );
+  })();
+
+  return (
+    <div className="grid place-items-center py-14 text-[#1D3C51]/80">
```

Comment:
@ap-justin @SovereignAndrey not sure how to handle this color `text-[#1D3C51]/80` that appears in Figma as the default text color, but it doesn't match any color we have in *tailwind.config.js*. Left it as defined in Figma until instructed to change to a different color or to add it to the config file.

Reply:
its not defined into their color variables but I'll check if it fits in the gray shades they provided - If not, we could just set is as lone color e.g `gray-page`

Reply:
Probably just a missed detail from the designer. We can fit it in, as Justin mentioned, to something matching or very close that exists the existing palette.

Reply:
`navy-l1` seems to be the closest color:
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/87bc3ae8-4946-412e-9f3b-db97a430f71c)
@ap-justin @SovereignAndrey we can use it for text instead of this Figma color.

Reply:
> `navy-l1` seems to be the closest color: ![image](https://private-user-

@misicnenad, If color is treated as blue(with 80 opacity), it could fit as `blue-d5`, But I am leaning on just treating it the way it looks `navy-l1`(NavyBlue-400 in design; intended for secondary text ) as @misicnenad suggested